modules/swagger-codegen/src/main/resources/java-sparkjava/model.mustache

```diff
@@ -0,0 +1,12 @@
+package {{package}};
```

Comment:
@giflw Would the model.mustache template in other Java server generator be a good starting point? I suggest we pick one and copy the file over.